tool/python/singa/model.py

```diff
@@ -325,7 +325,7 @@ def set_cudnn_layer_type(self, net):
             elif ly_type == kSoftmaxLoss: cudnn_ly_type = kCudnnSoftmaxLoss
             elif ly_type == kSTanh:
                 cudnn_ly_type = kCudnnActivation
-                net.layer[i].activation_conf.type = STANH
+                net.layer[i].activation_conf.type = TANH
```

Comment:
Agree,
for GPU
I keep activation="tanh" to TANH, and I will report error for activation="stahn"

By the way, for CPU
I changed  activation="tanh" to STANH, and I did NOT consider TANH.
Should I change to action="tanh" to TANH  and  activation="stanh" to STANH ?